Admin/AbstractAdmin.php

```diff
@@ -192,11 +193,22 @@
     /**
      * Whether or not to persist the filters in the session.
      *
+     * NEXT_MAJOR: remove this property
+     *
      * @var bool
+     *
+     * @deprecated since 3.x, to be removed in 4.0.
```

Comment:
We could use magic to trigger deprecation notices here, but since it's the admin class, which is already big, and other properties could be deprecated, this could become a maintenance nightmare, so I'd say leave it like that.